lib/game.rb

```diff
@@ -22,7 +22,7 @@ def col_select
       col = convert_input(input)
       if !["a", "b", "c", "d", "e", "f", "g"].include? input.downcase 
```

Comment:
Curious about these ! bangs

Reply:
I wanted to ask, "is this array MISSING this element?"  I tried finding an `<array>.exclude?` method, but it looks like that doesn't exist in Ruby by default so I went with `!<array>.include?`